src/javascript/lib/landing_pages/common.js

```diff
@@ -1,15 +1,17 @@
 // Check view width, add navbar height as offset if on desktop
 function checkWidth() {
-    let mq = window.matchMedia("(max-width: 1199px)");
+    var mq = window.matchMedia("(max-width: 1199px)");
```

Comment:
These js files will be transpiled [here](https://github.com/binary-com/binary-static/pull/3022).

Reply:
noted, thanks!